patches/minecraft/com/mojang/blaze3d/vertex/BufferBuilder.java.patch

```diff
@@ -10,6 +10,7 @@
 +      m_85722_(buffer.limit() + this.f_85658_.m_86020_());
 +      this.f_85648_.position(this.f_85654_ * this.f_85658_.m_86020_());
```

Comment:
This can be changed to just use `nextElementByte` instead of this math.

Reply:
Sure, changed it